app/views/charts/_form.html.haml

```diff
@@ -11,10 +11,12 @@
             = label_tag do
               = check_box_tag 'employment[past]', 1, params[:employment].try(:[], :past)
               Past
-          %span.checkbox-inline
-            = label_tag do
-              = check_box_tag 'employment[future]', 1, params[:employment].try(:[], :future)
-              Future
+          - @start_dates.each do |date|
```

Comment:
Creates "Future" check box for every employee starting in the future. Perhaps there's a way to do the calculation in the controller, and send something more abstract to the view template?